spring-cloud-gcp-bigquery/src/test/java/org/springframework/cloud/gcp/bigquery/integration/outbound/BigQueryFileMessageHandlerIntegrationTests.java

```diff
@@ -73,8 +73,9 @@ public void setup() {
 		this.taskScheduler.setPoolSize(1);
 		this.taskScheduler.initialize();
 
-		this.messageHandler = new BigQueryFileMessageHandler(this.bigquery, this.taskScheduler);
+		this.messageHandler = new BigQueryFileMessageHandler(this.bigquery);
 		this.messageHandler.setWriteDisposition(WriteDisposition.WRITE_TRUNCATE);
+		this.messageHandler.setTaskScheduler(this.taskScheduler);
```

Comment:
The `taskScheduler` must be destroyed in the end of test. Otherwise some orphan threads are going to stay yet.

Reply:
If I rely on the autowired instance through using `@EnableIntegration` then destroying it will not be necessary correct?

Reply:
Correct. That was a comment on outdated commit. Sorry for the noise. Although now you know what to do in test in the future anyway 😉